src/Sulu/Bundle/MarkupBundle/Markup/MarkupParser.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\MarkupBundle\Markup;
+
+use Sulu\Bundle\MarkupBundle\Tag\TagRegistryInterface;
+
+/**
+ * Parses html content and replaces special tags.
+ */
+class MarkupParser implements MarkupParserInterface
+{
+    const ATTRIBUTE_REGEX = '/(?<name>\b\w+\b)\s*=\s*"(?<value>[^"]*)"/';
+    const CONTENT_REGEX = '/(?:>(?<content>.*)<)/';
+    const TAG_REGEX = '/(?<tag><%s:(?<name>[a-z]*)[^\/>]*(?:\/>|>.*<\/%s:[^\/>]*>))/';
+
+    /**
+     * @var TagRegistryInterface
+     */
+    private $tagRegistry;
+    /**
+     * @var string
+     */
+    private $namespace;
+
+    /**
+     * @param TagRegistryInterface $tagRegistry
+     * @param string $namespace
+     */
+    public function __construct(TagRegistryInterface $tagRegistry, $namespace = 'sulu')
+    {
+        $this->tagRegistry = $tagRegistry;
+        $this->namespace = $namespace;
+    }
+
+    /**
+     * @param string $html
+     *
+     * @return string
+     */
+    public function parse($html)
```

Comment:
If we apply my previous suggestion `$html` should probably be renamed to `$html`.